pyvista/plotting/themes.py

```diff
@@ -161,7 +173,7 @@ def to_dict(self) -> dict:
         """
         # remove the first underscore in each entry
         dict_ = {}
-        for key in self.__slots__:
+        for key in self._all__slots__():
```

Comment:
This logic only really works because only one class in the hierarchy has non-empty `__slots__`.  It is needed because subclasses of `Theme` have no `__slots__`, we copy the attributes from the parent `Theme`.  A better solution in my opinion would be to get rid of these subclasses, i.e. make them objects of `Theme`, but this is an API change.